data/style/Grid.css

```diff
@@ -18,13 +18,13 @@
 .cell {
     background-color: @base_color;
     border-radius: 0;
-    border-top: solid 1px @menu_separator;
-    border-left: solid 1px @menu_separator;
+    border-top: solid 1px alpha (black, 0.15);
```

Comment:
I think I would prefer to add `menu_separator` to Greenfield so we can remain consistent

Reply:
Alright, reverting that.